beetle.gemspec

```diff
@@ -30,11 +30,13 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency "activesupport",           ">= 2.3.4"
 
   s.add_development_dependency "activerecord",        "~> 5.0"
+  s.add_development_dependency "pry-byebug",          "~> 3.5.0"
```

Comment:
Why did you feel this necessary? It's not related to the feature request and I don't want pry.

Reply:
✅

Reply:
oh I just needed this one to debug my setup, and I use it heavily in dev mode, but sure I remove it.